charts/prometheus/templates/prometheus-alerts-configmap.yaml

```diff
@@ -112,107 +121,803 @@ data:
+          record: instance:node_filesystem_usage:sum
+        - expr: sum(rate(node_network_receive_bytes_total[3m])) BY (instance)
+          record: instance:node_network_receive_bytes:rate:sum
+        - expr: sum(rate(node_network_transmit_bytes_total[3m])) BY (instance)
+          record: instance:node_network_transmit_bytes:rate:sum
+        - expr: sum(rate(node_cpu_seconds_total{mode!="idle",mode!="iowait"}[5m])) WITHOUT
+            (cpu, mode) / ON(instance) GROUP_LEFT() count(sum(node_cpu_seconds_total)
+            BY (instance, cpu)) BY (instance)
+          record: instance:node_cpu:ratio
+        - expr: sum(rate(node_cpu_seconds_total{mode!="idle",mode!="iowait"}[5m]))
+          record: cluster:node_cpu:sum_rate5m
+        - expr: cluster:node_cpu_seconds_total:rate5m / count(sum(node_cpu_seconds_total)
+            BY (instance, cpu))
+          record: cluster:node_cpu:ratio
+      - name: kube-prometheus-general-recording.rules 
+        rules:
+        - expr: count without(instance, pod, node) (up == 1)
+          record: count:up1
+        - expr: count without(instance, pod, node) (up == 0)
+          record: count:up0
+      - name: kube-state-metrics
+        rules:
+        - alert: KubeStateMetricsListErrors
+          annotations:
+            message: kube-state-metrics is experiencing errors at an elevated rate in
+              list operations. This is likely causing it to not be able to expose metrics
+              about Kubernetes objects correctly or at all.
+            runbook_url: 
+          expr: |
+            (sum(rate(kube_state_metrics_list_total{job="kube-state",result="error"}[5m]))
+              /
+            sum(rate(kube_state_metrics_list_total{job="kube-state"}[5m])))
+            > 0.01
+          for: 15m
+          labels:
+            severity: critical
             tier: platform
-            severity: urgent
+        - alert: KubeStateMetricsWatchErrors
           annotations:
-            summary: {{ printf "%q" "{{ $labels.pod_name }} ({{ $labels.container_name }}) is getting throttled {{ $value }}% of the time" }}
-            description: "In the past 5 minutes, one or more components in the platform namespace are experiencing throttling 80% or more of the time"
+            message: kube-state-metrics is experiencing errors at an elevated rate in
+              watch operations. This is likely causing it to not be able to expose metrics
+              about Kubernetes objects correctly or at all.
+            runbook_url: 
+          expr: |
+            (sum(rate(kube_state_metrics_watch_total{job="kube-state",result="error"}[5m]))
+              /
+            sum(rate(kube_state_metrics_watch_total{job="kube-state"}[5m])))
+            > 0.01
+          for: 15m
+          labels:
+            severity: critical
+            tier: platform
 
-        - alert: FluentdQueueLarge
-          expr: avg(fluentd_status_buffer_queue_length) > 2
+      - name: node-exporter
```

Comment:
I was discussing new group with Jim and where the alerts go. Jim indicated they are not routed based on the group name - they are routed based on the 'tier' label. so we are good to add new groups. https://github.com/astronomer/astronomer/blob/d3ccd4639373c4185d23e2024ee26dfaa37de169/charts/alertmanager/templates/alertmanager-configmap.yaml#L24